scripts/files/fs.py

```diff
@@ -117,13 +138,16 @@ def write_file(executor: ThreadPoolExecutor, input_: str, target: str) -> Future
     Returns:
         Future[str]: The result of the execution.
     """
-    future: Future[str] = Future()
     try:
-        future = executor.submit(write, os.path.join(target, f"{os.path.basename(input_)}"), read(input_))
+        get_log().info(f"Trying write from file: {input_}")
```

Comment:
Nit: In general it can be useful for readability to keep the contents of `try`/`except` blocks as small as possible. Since the logging code can't fail in any way relevant to this `try`/`except` block, it might be better to move it outside.